theia-docker/Dockerfile

```diff
@@ -9,10 +9,17 @@ RUN yarn && \
     yarn cache clean
 
 FROM node:8-alpine
+# See : https://github.com/theia-ide/theia-apps/issues/34
+RUN addgroup theia && \ 
+    adduser -G theia -s /bin/sh -D theia;
+RUN chmod g+rw /home && \
+    mkdir -p /home/project && \
+    chown theia:theia /home/project;    
 RUN apk add --no-cache git openssh bash
-WORKDIR /home/theia
+WORKDIR /home/project
```

Comment:
it should be `/home/theia`, otherwise Theia won't start, please rollback this line and check other images

after building the image, please always verify that it works with:
```bash
docker run -it -p 3000:3000 -v "$(pwd):/home/project:cached" theiaide/theia:next
```

I am getting the following error:
```bash
MacBook-Pro:theia kosyakov$ docker run -it -p 3000:3000 -v "$(pwd):/home/project:cached" theiaide/theia:next
yarn run v1.6.0
$ /home/project/node_modules/.bin/theia start /home/theia --hostname=0.0.0.0
module.js:549
    throw err;
    ^

Error: Cannot find module '/home/project/src-gen/backend/main.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:612:3
Done in 1.33s.
```

Reply:
Whoops, I shouldn't change that, will fix shortly

Reply:
Re-squashed and fixed it